### DISCUSSION TITLE
Proofreading the interface

### DISCUSSION THREAD

**Discussion by timtfj** [mastodon/mastodon#22514] [Q&A]
Among other things, my background includes a lot of proofreading and copy-editing. So every so often, I notice small typos and the like in the web interface, that could do with correcting.

Is https://github.com/mastodon/mastodon/issues a suitable place to report these, or is there somewhere else that's dedicated to that?

Thanks in advance,
Tim.

**Comment by mjankowski** [comment]
I realize this is three years old, but -- yes, making issues is fine.

If it's truly a small (ie, one short phrase/word/sentence) change and you are comfortable doing so, make a PR directly. Otherwise issue is good.
